### DISCUSSION TITLE
It dont have uninstall/remove subcommand, how to uninstall installed components?

### DISCUSSION THREAD

**Discussion by shishirong** [miurahr/aqtinstall#805] [Q&A]
just delete folder?

**Comment by pzhlkj6612** [comment]
Hi! aqtinstall does not have the ability to remove components. Directly deleting folders might be OK.

Are you going to free up some disk space? Or you've encountered some problems with the Qt installation? What components have you installed?

**Reply by shishirong** [reply]
for example:  I installed example use `aqt install-example -O "C:/Qt/" windows desktop 5.15.2 -m all`， How can I uninstall those example?

**Reply by pzhlkj6612** [reply]
> for example: I installed example use `aqt install-example -O "C:/Qt/" windows desktop 5.15.2 -m all`， How can I uninstall those example?

I've tried your command and there is only one top-level directory created:

```console
$ aqt install-example -O "Qt" windows desktop 5.15.2 -m all
...

$ find Qt -maxdepth 3
Qt
Qt/Examples
Qt/Examples/Qt-5.15.2
Qt/Examples/Qt-5.15.2/assistant
Qt/Examples/Qt-5.15.2/vulkan
Qt/Examples/Qt-5.15.2/opengl
Qt/Examples/Qt-5.15.2/delayedencoding.qdoc
...

```

Therefore, you can safely delete the "Examples/Qt-5.15.2" directory directly.

<br>

Aqtinstall does not know the structure of all Qt files. It just downloads the packaged Qt files uploaded by Qt staff [^1]. To avoid accidentally breaking your Qt installation, I think you'd better download new Qt files to a new place and then remove the old one.

[^1]: https://download.qt.io/online/qtsdkrepository/windows_x86/desktop/qt5_5152_src_doc_examples/
